.nvmrc

```diff
@@ -0,0 +1 @@
+v16
```

Comment:
I think we can switch it to the latest LTS, Node.js 18

Reply:
Sure, but are you referring to the .nvmrc file or the Contribution guide?
Usually, you should state the min Node version, but then .nvmrc should match that version. What do you think?

Reply:
I think we can notice that the minimum version is v16, but we recommend that v18 is better for the developers.

And set `.nvmrc` to v18.